autonomy/cli/helpers/chain.py

```diff
@@ -374,10 +373,6 @@ def mint_component(
             raise click.ClickException(
                 f"Component mint failed with following error; {e}"
             ) from e
-        except FailedToRetrieveTokenId as e:
-            raise click.ClickException(
-                f"Component mint was successful but token ID retrieving failed with following error; {e}"
-            ) from e
 
```

Comment:
Why are these exceptions removed?

Reply:
https://github.com/valory-xyz/open-autonomy/pull/2050#discussion_r1322411641 It's not required anymore since we're not going to use polling for checking if the component was minted or not